### DISCUSSION TITLE
Last pool time for Libreview

### DISCUSSION THREAD

**Discussion by filipsawczenko** [j-kaltes/Juggluco#3] [Ideas]
Hi all,

I find useful to have an info of time when last data pool  was sent to Libreview. Now it's necessary to dig into settings- maybe it could be somewhere on main screen? Or at least a warning message that some time passed from last successful upload. It is mandatory to have all data in LV to have prescription for FSL2 in Poland. 

Cheers
Filip

**Comment by j-kaltes** [comment]
Are you serious?
Every time you switch away from Juggluco, by going to another app or turning of the screen, data is send to Libreview when it wasn't done in the last 20 minutes. So what is the problem? Touching three times on the screen to see when it was last send is also no large effort. 
Are you saying that in Poland you are obliged to be constantly connected to the internet wherever you go, because otherwise you have to pay yourself for your Freestyle Libre 2 sensors yourself?
How many times a day do you otherwise have to scan?

**Reply by filipsawczenko** [reply]
Well, 
In general in my case Juglucco doesn't send data automatically every time- when there is no rush on phone- at night it doesn't do this at all (Samsung galaxy note 10 lite, Android 13). I've made all permissions for app. Only action to do this is to scan sensor via NFC, or press "send data". 
Yes, in PL we have this kind of agreement. You have to have min 75% of sensor activity in LV reported. Otherwise we loose prescription. 
To be clear- I appreciate Your effort to build this app, and I think it has great features for us. My request was not just to look for weaknesses or whatever - just to upgrade. Don't think I am lazy troll please. 

BR
F.

**Reply by j-kaltes** [reply]
When you use Librelink, the values are only sent to Libreview when you scan at least once every 8 hours. Now you need to view Juggluco at least once every 8 hours to make that all glucose readings are send to Libreview. With Left menu->Settings->Libreview you can see the time of the last sync.
